quilt/tools/build.py

```diff
@@ -178,10 +226,28 @@ def build_package_from_contents(username, package, build_dir, data):
     if pkgformat is PackageFormat.HDF5:
         raise BuildException("HDF5 format is no longer supported; please use PARQUET instead.")
 
+    # inline checks take precedence
+    if 'checks' in build_data:
+        if checks_contents is None:
+            checks_contents = build_data['checks']
+        else:
+            checks_contents.update(build_data['checks'])
+
     store = PackageStore()
-    newpackage = store.create_package(username, package)
-    _build_node(build_dir, newpackage, '', contents, pkgformat)
-    newpackage.save_contents()
+    if dryrun:
+        from .core import RootNode
+        from .package import Package
+        newpackage = Package(
```

Comment:
Might be cleaner to add a dry-run option to `store.create_package`

Reply:
DONE